internal/surface/backend/validate_internal_test.go

```
@@ -0,0 +1,368 @@
+// The external version of this test cannot: it can only loop to the last
+// *named* constant it can see, so appending a constant and forgetting its
+// name-table entry leaves the loop stopping short — the array grows with an
+// empty default, String returns "", and the test written to catch exactly that
+// passes. Only an in-package test can reach the sentinel, which is where the
+// real bound lives.
+func TestClosedEnums_EveryConstantIsNamed(t *testing.T) {
+	enums := map[string]struct {
+		count int
+		name  func(int) string
+	}{
+		"Kind":              {int(kindCount), func(i int) string { return Kind(i).String() }},
```

golangci-lint (2.12.2)

[high] G115: integer overflow conversion int -> uint8

(gosec)